haskell/Makefile

```diff
@@ -20,11 +20,15 @@ PACKAGE = stablecoin
 #   Initial `../..` in `ln` is needed because of `test/resources`.
 #   $(LIGO_DIR) is relative to the current directory.
 define call_test
+	$(call build_stablecoin_contract,"","")
```

Comment:
AFAIU `"",""` things here are like functions arguments. You can use `$1` and `$2` there. But apparently you don't use them. So why do you pass them?

Reply:
copypaste is evil